src/Agent.Worker/TestResults/TestResultsServer.cs

```diff
@@ -23,14 +24,14 @@ public class TestResultsServer : AgentService, ITestResultsServer
     {
         private VssConnection _connection;
 
-        private TestManagementHttpClient TestHttpClient { get; set; }
+        private TestResultsHttpClient TestHttpClient { get; set; }
 
         public void InitializeServer(VssConnection connection)
         {
             ArgUtil.NotNull(connection, nameof(connection));
             _connection = connection;
 
-            TestHttpClient = connection.GetClient<TestManagementHttpClient>();
+            TestHttpClient = connection.GetClient<TestResultsHttpClient>();
```

Comment:
you might want to do it under FF same way we have done in other places

            FeatureAvailabilityHttpClient featureAvailabilityHttpClient = connection.GetClient<FeatureAvailabilityHttpClient>();

            if (FeatureFlagUtility.GetFeatureFlagState(featureAvailabilityHttpClient, CodeCoverageConstants.EnablePublishToTcmServiceDirectlyFromTaskFF, context))

            {

                TestResultsHttpClient tcmClient = connection.GetClient<TestResultsHttpClient>();


            }

            else

            {

                TestManagementHttpClient tfsClient = connection.GetClient<TestManagementHttpClient>();



            }

Reply:
Changed it to be consistent. Need to remove, when we get away with the FF.